pkgs/tools/security/blindelephant/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, python2, fetchsvn }:
+
+python2.pkgs.buildPythonApplication rec {
+  pname = "blindelephant";
+  version = "7";
+
+  src = fetchsvn {
+    url = "svn://svn.code.sf.net/p/blindelephant/code/";
+    rev = version;
+    sha256 = "0ji5dk47643mdhkg05g9mwh6pnjqms71f5lng6wjhjr4v5h57hiz";
+  };
+
+  preConfigure = ''
```

Comment:
Does setting `sourceRoot = "source/trunk/src";` or some similar path also works instead of changing the root path manually?

Reply:
Tested it and it works, thanks!